packages/camera/camera_platform_interface/lib/src/platform_interface/camera_platform.dart

```diff
@@ -96,11 +96,10 @@ abstract class CameraPlatform extends PlatformInterface {
     throw UnimplementedError('onCameraTimeLimitReached() is not implemented.');
   }
 
-  /// The device orientation changed.
+  /// The ui orientation changed.
```

Comment:
?

Reply:
Same explanation as the previous comment. It was decided to not rename the interface method to prevent making a breaking change.